include/seqan3/io/record.hpp

```diff
@@ -36,76 +36,112 @@ namespace seqan3
+    ID SEQAN3_DEPRECATED_310,             //!< Please use the field name in lower case.
+    QUAL SEQAN3_DEPRECATED_310,           //!< Please use the field name in lower case.
+    SEQ_QUAL SEQAN3_DEPRECATED_310,       //!< Please use the field name in lower case.
+    OFFSET SEQAN3_DEPRECATED_310,         //!< Please use the field name in lower case.
+    BPP SEQAN3_DEPRECATED_310,            //!< Please use the field name in lower case.
+    STRUCTURE SEQAN3_DEPRECATED_310,      //!< Please use the field name in lower case.
+    STRUCTURED_SEQ SEQAN3_DEPRECATED_310, //!< Please use the field name in lower case.
+    ENERGY SEQAN3_DEPRECATED_310,         //!< Please use the field name in lower case.
+    REACT SEQAN3_DEPRECATED_310,          //!< Please use the field name in lower case.
+    REACT_ERR SEQAN3_DEPRECATED_310,      //!< Please use the field name in lower case.
+    COMMENT SEQAN3_DEPRECATED_310,        //!< Please use the field name in lower case.
+    ALIGNMENT SEQAN3_DEPRECATED_310,      //!< Please use the field name in lower case.
+    REF_ID SEQAN3_DEPRECATED_310,         //!< Please use the field name in lower case.
+    REF_SEQ SEQAN3_DEPRECATED_310,        //!< Please use the field name in lower case.
+    REF_OFFSET SEQAN3_DEPRECATED_310,     //!< Please use the field name in lower case.
+    HEADER_PTR SEQAN3_DEPRECATED_310,     //!< Please use the field name in lower case.
+    FLAG SEQAN3_DEPRECATED_310,           //!< Please use the field name in lower case.
+    MATE SEQAN3_DEPRECATED_310,           //!< Please use the field name in lower case.
+    MAPQ SEQAN3_DEPRECATED_310,           //!< Please use the field name in lower case.
+    CIGAR SEQAN3_DEPRECATED_310,          //!< Please use the field name in lower case.
+    TAGS SEQAN3_DEPRECATED_310,           //!< Please use the field name in lower case.
+    BIT_SCORE SEQAN3_DEPRECATED_310,      //!< Please use the field name in lower case.
+    EVALUE SEQAN3_DEPRECATED_310,         //!< Please use the field name in lower case.
+    USER_DEFINED_0 SEQAN3_DEPRECATED_310, //!< Please use the field name in lower case.
+    USER_DEFINED_1 SEQAN3_DEPRECATED_310, //!< Please use the field name in lower case.
+    USER_DEFINED_2 SEQAN3_DEPRECATED_310, //!< Please use the field name in lower case.
+    USER_DEFINED_3 SEQAN3_DEPRECATED_310, //!< Please use the field name in lower case.
+    USER_DEFINED_4 SEQAN3_DEPRECATED_310, //!< Please use the field name in lower case.
+    USER_DEFINED_5 SEQAN3_DEPRECATED_310, //!< Please use the field name in lower case.
+    USER_DEFINED_6 SEQAN3_DEPRECATED_310, //!< Please use the field name in lower case.
+    USER_DEFINED_7 SEQAN3_DEPRECATED_310, //!< Please use the field name in lower case.
+    USER_DEFINED_8 SEQAN3_DEPRECATED_310, //!< Please use the field name in lower case.
+    USER_DEFINED_9 SEQAN3_DEPRECATED_310, //!< Please use the field name in lower case.
```

Comment:
Do we really need this?

Reply:
Well since this is actually already used rather often, I think it is important to provide a clear message to the user. Additionally this does not hurt much, does it?

Reply:
@rrahn Do you agree?

Reply:
yes it is fine.